src/main/java/com/daml/extensions/testing/utils/PackageUtils.java

```diff
@@ -188,6 +190,11 @@ public static DamlLf1.Package findPackageObject(
       throws InvalidProtocolBufferException {
     PackageClient pkgClient = ledgerClient.getPackageClient();
     String pkgId = findPackage(ledgerClient, moduleName);
+    try {
```

Comment:
Do we need these sleeps? Can we maybe at least parameterize them?
```
sleep(SOME_CONSTANT_USED_EVERYWHERE);
```

Reply:
these sleeps helped me to avoid PACKAGE_NOT_FOUND problem. Now I see that time after time it is still happen - it cannot find package even after short sleep. So what I'm gonna to do now is to add "waitForPackage" procedure. That will do several attempts to fetch the package

Reply:
good idea